<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CPU Usage</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>173</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblCPUS">
       <property name="font">
        <font>
         <family>Liberation Sans</family>
         <pointsize>11</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(51, 255, 238);
background-color: rgb(6, 37, 48);
border-style: solid;
border-width: 2px; /* Sets the border thickness to 2 pixels */
border-color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>Number of CPUs:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCPUSValue">
       <property name="font">
        <font>
         <family>Liberation Sans</family>
         <pointsize>11</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>141</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="HorizontalCore">
     <item>
      <layout class="QVBoxLayout" name="VerticalCore"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="VerticalLoad"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
